AK/UUID.cpp

```diff
@@ -109,7 +109,7 @@ DeprecatedString UUID::to_deprecated_string() const
     builder.append(encode_hex(m_uuid_buffer.span().slice(8).trim(2)).view());
     builder.append('-');
     builder.append(encode_hex(m_uuid_buffer.span().slice(10).trim(6)).view());
```

Comment:
For this line and the above, i think builder has a try_append

(Also I am sure there is a better way to slice with an offset than span-slice-trim-view)

Reply:
Changed to `try_append`, modified the code to be _somewhat_ better to look at. This function is not used at all in our codebase by the way :^)